src/components/QueryOutput/QueryOutput.js

```diff
@@ -1865,16 +1865,25 @@ export default class QueryOutput extends React.Component {
   }
 
   renderHTMLMessage = (queryResponse) => {
-    return (
-      <HTMLRenderer
-        html={_get(queryResponse, 'data.data.answer')}
-        components={{
-          a: (props) => {
-            return <a {...props} target="_blank" />
-          },
-        }}
-      />
-    )
+    if (_get(queryResponse, 'data.data.answer') !== null) {
```

Comment:
You should default value this like _get(queryResponse, 'data.data.answer', null) !== null

Unless you know that it will have a value or only be null

Reply:
fixed